app/questionnaire/rules.py

```diff
@@ -164,6 +169,14 @@ def evaluate_repeat(schema, repeat_rule, answer_store, answer_ids_on_path):
 
         answers = list(answer_store.filter(answer_ids=repeat_indexes))
 
+        # Get answer block_id
+        # Use answer group_instance
+        # is block_id / group_instance on routing path?
```

Comment:
This block of comments still needed?  If so, putting them above the line that does the action would be more useful